### DISCUSSION TITLE
Inconsistent color tokens for text/icons

### DISCUSSION THREAD

**Discussion by dholliday3** [twilio-labs/paste#2586] [Q&A] [answered]
Hello, I'm using a color token for an icon and noticed that the rgb value in the documentation is different from what I'm getting in the browser. 

**Here's what I'm expecting:** 
- $color-text-icon-warning 
- rgb(255, 179, 122) 
![image](https://user-images.githubusercontent.com/15958912/182739060-4ba8e094-aabc-4429-8cb7-5dd18c59f5c7.png)

**What I have in my code and what I got:** 
- `<WarningIcon decorative={true} color="colorTextIconWarning" />`
- rgb(227, 106, 25)
![image](https://user-images.githubusercontent.com/15958912/182739122-0af477bd-92fa-4f25-b3af-81f279aac3e0.png)

Am I missing something? I'm looking for a yellow warning icon but using that token isn't giving me the expected RGB values. 
Thanks!

**Comment by loreina** [comment, marked as answer]
Hi @dholliday3,

Our token values change with the theme you're using (default or dark) to serve multiple design languages and comply with [accessibility standards](https://paste.twilio.design/foundations/colors#wcag-guidelines-and-color).

When using the light/dark toggle on the Paste docs site, the token list page will swap to either show default theme values with light mode, or dark theme values in dark mode. So in default theme, `colorTextIconWarning` points to rgb(227, 106, 25) and in dark theme, `colorTextIconWarning` points to rgb(255, 179, 122). 

Hope that clarifies things. Feel free to reply here if you have any other questions!

**Reply by dholliday3** [reply]
Gotcha, thanks for responding! 

Just as feedback I think it might be helpful to make it more obvious that picking dark/light mode on the docs changes the color tokens as well! My teammates were also surprised by this design as this isn't a pattern we've seen in other docs. 

Thanks for the prompt responses!

**Reply by serifluous** [reply]
Thanks for the feedback! This is actually super great timing because we've been working on some updates to our tokens page that just went live this morning. Check it out and let us know if this is what you were expecting: https://paste.twilio.design/tokens/list
